hbase-server/src/main/resources/hbase-webapps/master/snapshot.jsp

```diff
@@ -99,6 +100,7 @@
           <% } %>
         </td>
         <td><%= new Date(snapshot.getCreationTime()) %></td>
+        <td><%= snapshot.getTtl() %></td>
```

Comment:
Can print "FOREVER" here or something similar if value is 0, so the user is less likely to be confused